client/src/app/app.module.ts

```diff
@@ -1,14 +1,37 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import {AbsenceFormComponent} from './components/form/form.component';
+import { ReactiveFormsModule } from '@angular/forms';
+import { NgxErrorsModule } from '@ultimate/ngxerrors';
+import { TimepickerModule } from 'ngx-bootstrap';
+import { BsDatepickerModule } from 'ngx-bootstrap';
+import {DashboardComponent} from './containers/dashboard/dashboard.component';
+import {FormsModule} from '@angular/forms';
 
 import { AppComponent } from './app.component';
 
+
+const appRoutes: Routes = [
```

Comment:
Move route configuration to separate file, check this [repo](https://github.com/kucharskimaciej/dm-ng2-solutions/tree/Routes10.4/src/app). See the `app.routes.ts`